src/interface/can_translation/include/can_translation/types.hpp

```diff
@@ -0,0 +1,19 @@
+/*
+ * Package:   can_translation
+ * Filename:  include/can_translation/types.hpp
+ * Author:    Joshua Williams
+ * Email:     joshmackwilliams@protonmail.com
+ * Copyright: 2021, Nova UTD
+ * License:   MIT License
+ */
+
+#pragma once
+
+namespace navigator {
+namespace can_translation {
+
+typedef uint32_t can_id_t;
```

Comment:
Why separate typedefs into another hpp?

Reply:
The reason is that they don't really belong in any other header file. The `can_translation` package may grow over time, and these types will be relevant to the new nodes even though the other headers are not. And, I'd rather avoid duplicating them, so that they can be changed easily.